### DISCUSSION TITLE
Build error on build server: wix.exe : error WIX0217

### DISCUSSION THREAD

**Discussion by forrma** [wixtoolset/issues#7518] [Questions]
When building my WiX 4 setup project on the build server, the following error occurs:

`wix.exe : error WIX0217: Error executing ICE action 'ICE04'. The following string format was not expected by the external UI message logger: "The Windows Installer Service could not be accessed. This can occur if the Windows Installer is not correctly installed. Contact your support personnel for assistance.`

For the build msbuild is used:

`msbuild MyProduct.Setup.sln /property:Configuration=Release /property:SoftwareVersion=%SoftwareVersion% -p:AssemblyVersion=%SoftwareVersion% -p:Version=%SoftwareVersion% -p:FileVersion=%SoftwareVersion%`

Thanks for tips

**Comment by BMurri** [comment]
Your build server is running a non-interactive login that is not elevated. Unfortunately, that means that the Windows Installer service is not accessible, and MSI validation currently requires access to that service. You'll need to suppress validation during your build and run it separately in a different environment.

**Reply by forrma** [reply]
Thank you for your answer.

How can I suppress the validation?

**Reply by BMurri** [reply]
Based on a quick glance at the [docs](https://wixtoolset.org/docs/intro/):

If using [`MSBuild`/`Visual Studio`](https://wixtoolset.org/docs/tools/msbuild/) set the property `SuppressValidation` to true, if using [`wix`/`dotnet wix`](https://wixtoolset.org/docs/tools/wixexe/) do not include `wix msi validate`.
